packages/fast-development-site-react/app/index.html

```diff
@@ -4,6 +4,7 @@
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width,initial-scale=1">
         <title>FAST React development app</title>
+        <link rel="shortcut icon" href="../src/assets/images/favicon.ico">
```

Comment:
we wouldn't want to source this from "./src" because it won't be bundled when webpack builds.

Reply:
+1